### DISCUSSION TITLE
How to check duplicate submissions by email

### DISCUSSION THREAD

**Discussion by laurabennett** [solspace/craft-freeform#600] [Feature Requests]
I would like users to only be able to submit a form once per email address. Is there a simple way to do this?

**Comment by kjmartens** [comment]
Unfortunately there isn't a way to do this at this time.

Currently we just offer the following duplicate checks on submissions:
https://docs.solspace.com/craft/freeform/v4/overview/submission-limits.html#per-user

- Logged in Users only (no limit)
- Once per Cookie only
- Once per IP/Cookie combo
- Once per logged in Users only
- Once per logged in User or Guest Cookie only
- Once per logged in User or Guest IP/Cookie combo

**Reply by laurabennett** [reply]
This ended up being a deal breaker for us and Formie was the solution.
Thanks

**Reply by kjmartens** [reply]
Sorry about that @laurabennett. Just wanted to let you know we plan on implementing this in a future version of Freeform. 🙂

**Comment by kjmartens** [comment]
This is now available in Freeform 4.0.24+. 🙂
